conllu/serializer.py

```diff
@@ -21,8 +21,9 @@ def serialize_field(field: T.Any) -> str:
             if value == "":
                 fields.append(key)
                 continue
-
-            fields.append('='.join((key, value)))
+            if any(isinstance(value, serializable_type) for serializable_type in (int, float, str, bool)):
```

Comment:
That would indeed be simpler. My reason for not doing this was that I've generally been taught to use _isinstance_ instead of _type(x) ==_ to check types, for the reason that this respects inheritance. For example, if a user were to be using an exotic subclass of _int_, it would still be recognised.

However, I think this would be an extreme edge cases for these types, so the simpler syntax may be preferable.

Reply:
Great motivation. TokenList is itself a subclass of list, so I think it makes sense to check subclasses too like you do.